packages/webpack/README.md

```diff
@@ -92,7 +92,16 @@ Using the experimental `--experimental-esbuild` option will replace the `babel-l
 
 To use it, install [`esbuild-loader`](https://www.npmjs.com/package/esbuild-loader) and [`esbuild-jest`](https://www.npmjs.com/package/esbuild-jest) as dev-dependencies in your project.
 
-Please report any bugs you may encounter.
+In order to use esbuild in Jest you need to set the `USE_EXPERIMENTAL_ESBUILD` environment variable to `true`.
+
+**Please be aware that things are working differently in esbuild than Babel. The currently known drawbacks and limitations are:**
+
+- esbuild is not on a stable release cycle yet. Please also read about the [production readyness](https://esbuild.github.io/faq/#production-readiness) from the main author.
+- esbuild does not typecheck your TypeScript files, it can only convert them to JavaScript.
+- to use the JSX syntax in TypeScript you have to use the `.tsx` extention. We recommend to also use the `.jsx` extention for JavaScript based JSX files to be consistent, even though it's not required.
+- Hops' `importComponent` is currently a Babel plugin and we only support the simple syntax (arrow function with import: `() => import(...)`) in esbuild mode for now.
+
```

Comment:
I'd add a code block here demonstrating a valid use of `importComponent` and an invalid one.